### PR TITLE
Create dynamic Calendar module

### PR DESCRIPTION
**Pull request by juuprz**
Hi Guys - had to edit a piece in my circle CI to get the automated testing running correctly.

### CODE REVIEW

File: client/components/BookingContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    $.ajax({
+      url: `${dummyUserRoomPath}`,
+      method: 'GET'
+    })
+      .done((res) => {
+        res = JSON.parse(res);
+        console.log(res.availabilityScore);
+        this.modelAvailabilityForSelectedRoom(res.availabilityScore);
+      });
+  }
```
Comment (Lamanova) [inline client/components/BookingContainer.jsx]: We want the static file to be served only for the path ‘/rooms/:id/‘. So you can state that the static file will be served only for that path (app.use('/rooms/:id/', express.static(path.join(__dirname, <path to your index.html file>)))

Comment (Lamanova) [reply client/components/BookingContainer.jsx]: You can get room number from window.location.pathname and pass it as props to the Booking component

**Review by Lamanova — CHANGES_REQUESTED** [review]

File: client/components/BookingContainer.jsx (hunk 1 above)
Comment (juuprz) [reply client/components/BookingContainer.jsx]: Hi @Lamanova requested changes have been made. Please check & provide approval.

**Review by Lamanova — APPROVED** [review]

**Comment by Lamanova** [comment]
Unique 'key' prop error has to be fixed

**Pull request merged**